### PR TITLE
[ODS-4274] Convert EdFi.Ods.Api.IntegrationTestHarness to .NET Core

### PR DESCRIPTION
**Pull request by raghuramakrishna**

### CODE REVIEW

File: Application/EdFi.Ods.Api.Common/Constants/ApiFeature.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@ public class ApiFeature : Enumeration<ApiFeature, string>
         public static readonly ApiFeature OwnershipBasedAuthorization = new ApiFeature("ownershipBasedAuthorization", "Ownership Based Authorization");
         public static readonly ApiFeature UniqueIdValidation = new ApiFeature("uniqueIdValidation", "Unique Identification Validation");
         public static readonly ApiFeature TokenInfo = new ApiFeature("tokenInfo", "Token Introspective Endpoint");
+        public static readonly ApiFeature UpdateAdminDatabase = new ApiFeature("updateAdminDatabase", "Update Admin Database");
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Api.Common/Constants/ApiFeature.cs:21]: this needs to be removed its not a feature we need to support

File: Application/EdFi.Ods.Api.NetCore/Startup/OdsStartupBase.cs [hunk 2]
```diff
@@ -48,17 +48,17 @@ public abstract class OdsStartupBase
...
-            var builder = new ConfigurationBuilder()
-                .SetBasePath(env.ContentRootPath)
-                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
-                .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
-                .AddEnvironmentVariables();
+            //var builder = new ConfigurationBuilder()
+            //    .SetBasePath(env.ContentRootPath)
+            //    .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+            //    .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
+            //    .AddEnvironmentVariables();
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Api.NetCore/Startup/OdsStartupBase.cs:59]: remove the comment code please

File: Application/EdFi.Ods.Api.NetCore/Startup/OdsStartupBase.cs [hunk 3]
```diff
@@ -48,17 +48,17 @@ public abstract class OdsStartupBase
...
 
-            Configuration = builder.Build();
+            Configuration = (IConfigurationRoot)configuration;
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Api.NetCore/Startup/OdsStartupBase.cs:61]: please fix formatting there should be a space after the cast

File: Application/EdFi.Ods.Api.NetCore/Startup/OdsStartupBase.cs [hunk 4]
```diff
@@ -48,17 +48,17 @@ public abstract class OdsStartupBase
         private const string CorsPolicyName = "_development_";
         private readonly ILog _logger = LogManager.GetLogger(typeof(OdsStartupBase));
 
-        public OdsStartupBase(IWebHostEnvironment env)
+        public OdsStartupBase(IWebHostEnvironment env, IConfiguration configuration)
         {
             _logger.Debug("Loading configuration files");
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Api.NetCore/Startup/OdsStartupBase.cs:53]: this debug statement would be moved to the program.cs file

Comment (raghuramakrishna) [reply Application/EdFi.Ods.Api.NetCore/Startup/OdsStartupBase.cs:53]: Done

**Pull request merged**